frontend/src/components/Sidepanel/Sidepanel.tsx

```diff
@@ -45,12 +48,25 @@ const panels: PanelItem[] = [
 const Sidepanel = () => {
   const [activePanel, setActivePanel] = useState<PanelItem>()
 
+  const projectType = useProjectStore(s => s.project.config.type)
+
   // Open panel via event
   useEvent('sidepanel:open', e => {
     const panel = panels.find(p => p.value === e.detail.panel)
     setActivePanel(activePanel?.value === panel.value ? undefined : panel)
   }, [activePanel])
 
+  // Show bottom panel with TM Tape Lab
+  useEffect(() => {
+    if (projectType === 'TM') {
+      if (activePanel?.value === 'test') {
+        dispatchCustomEvent('bottomPanel:open', { panel: 'tmTape' })
+      } else {
+        dispatchCustomEvent('bottomPanel:close', null)
+      }
+    }
```

Comment:
You might as well merge them into a single statement
```suggestion
    if (projectType === 'TM' && activePanel?.value === 'test') {
      dispatchCustomEvent('bottomPanel:open', { panel: 'tmTape' })
    } else {
      dispatchCustomEvent('bottomPanel:close', null)
    }
```
Also this reminds me, currently the BottomPanel isn't actual rendered for other project types (`Editor.tsx:99` has a statement that stops it)